kernel_corepattern.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package procfs
+
+import (
+	"bytes"
+	"os"
+
+	"github.com/prometheus/procfs/internal/util"
+)
+
+// KernelCorePattern returns value from /proc/sys/kernel/core_pattern
+func (fs FS) KernelCorePattern() (pattern []byte, err error) {
```

Comment:
Don't name these returns.

```suggestion
func (fs FS) KernelCorePattern() ([]byte, error) {
```